sunbird-mentoring/package.json

```diff
@@ -10,6 +10,7 @@
 	"license": "ISC",
 	"dependencies": {
 		"axios": "^1.4.0",
+		"dotenv": "^16.4.5",
```

Comment:
@rakeshSgr don't we have to add requiredDependencies here?

Reply:
not require